push-notifications/README.md

```diff
@@ -68,6 +68,19 @@ An empty Array can be provided if none of the previous options are desired. `pus
 }
 ```
 
+## Silent Push Notifications / Data-only Notifications
+#### iOS
+This plugin does not support iOS Silent Push (Remote Notifications).  We recommend using native code solutions for handling these types of notifications, see [Pushing Background Updates to Your App](https://developer.apple.com/documentation/usernotifications/setting_up_a_remote_notification_server/pushing_background_updates_to_your_app).
+
+#### Android
+This plugin does support data-only notifications, but will NOT call `pushNotificationReceived` if the app has been killed.  To handle this scenario, you will need to create a service that extends `FirebaseMessagingService`, see [Handling FCM Messages](https://firebase.google.com/docs/cloud-messaging/android/receive). 
+
+## Common Issues
+On Android, there are various system and app states that can affect the delivery of push notifications:
+
+* If the device has entered [Doze](https://developer.android.com/training/monitoring-device-state/doze-standby) mode, your application may have restricted capabilities.  To increase the chance of your notification being received, consider using [FCM high priority messages](https://firebase.google.com/docs/cloud-messaging/concept-options#setting-the-priority-of-a-message).
+* There are differences in behavior between development and production.  Try testing your app outside of being launched by Android Studio.  Read more [here](https://stackoverflow.com/questions/48642423/firebase-push-notification-issue-in-android-when-app-is-closed-killed).
```

Comment:
```suggestion
* There are differences in behavior between development and production. Try testing your app outside of being launched by Android Studio.  Read more [here](https://stackoverflow.com/questions/48642423/firebase-push-notification-issue-in-android-when-app-is-closed-killed).
```
another one

Reply:
I personally don't like linking SO threads, they can get out of date, deleted, etc.
But in any case, it would be better to link the answer directly rather than the question `https://stackoverflow.com/a/50238790/1351469`